build-system/tasks/helpers.js

```diff
@@ -360,11 +358,18 @@ function compileUnminifiedJs(srcDir, srcFilename, destDir, options) {
     fast: true,
     ...options.browserifyOptions,
   };
-
-  let bundler = browserify(browserifyOptions).transform(babelify, {
+  const babelifyOptions = {
     caller: {name: 'unminified'},
     global: true,
-  });
+  };
+  if (argv.coverage) {
+    babelifyOptions['plugins'] = ['babel-plugin-istanbul'];
+  }
+
+  let bundler = browserify(browserifyOptions).transform(
+    babelify,
+    babelifyOptions
+  );
```

Comment:
This entire block should be reverted. Babel (`babelify`) automatically picks up its config from `babel.config.js`, which loads the correct set of plugins based on `caller`.

Reply:
Done.